--- Term-ReadLine-Gnu-1.17/eg/fileman~	1998-02-28 10:01:34.000000000 +0000
+++ Term-ReadLine-Gnu-1.17/eg/fileman	2008-03-02 16:27:50.629026340 +0000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # $Id$
 #
--- Term-ReadLine-Gnu-1.17/eg/perlsh~	2005-01-20 05:24:21.000000000 +0000
+++ Term-ReadLine-Gnu-1.17/eg/perlsh	2008-03-02 16:27:59.628407833 +0000
@@ -1,4 +1,4 @@
-#! /usr/local/bin/perl
+#!/usr/bin/perl
 #
 #	$Id$	
 #
--- Term-ReadLine-Gnu-1.17/eg/pftp~	1999-03-19 17:46:02.000000000 +0000
+++ Term-ReadLine-Gnu-1.17/eg/pftp	2008-03-02 16:28:02.128236026 +0000
@@ -1,4 +1,4 @@
-#! /usr/local/bin/perl
+#!/usr/bin/perl
 #
 #	$Id$
 #
--- Term-ReadLine-Gnu-1.17/Gnu/XS.pm~	2008-02-07 15:44:08.000000000 +0000
+++ Term-ReadLine-Gnu-1.17/Gnu/XS.pm	2008-03-02 16:29:48.620917026 +0000
@@ -1,4 +1,3 @@
-#!/usr/local/bin/perl
 #
 #	XS.pm : perl function definition for Term::ReadLine::Gnu
 #
--- Term-ReadLine-Gnu-1.17/eg/ptksh+.orig	2008-03-02 16:50:56.533776262 +0000
+++ Term-ReadLine-Gnu-1.17/eg/ptksh+	2008-03-02 16:50:16.036559543 +0000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl
 #
 # $Id$
 #
